src/index.html

```diff
@@ -22,6 +22,370 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__navigation navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a
+              class="navigation__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navigation__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="navigation__item">
```